client/src/components/ReviewList.jsx

```diff
@@ -26,6 +35,18 @@ class ReviewList extends React.Component {
     })
   }
 
+  getReviews(product_id) {
+    const params = new URLSearchParams({ sort: "newest", product_id: product_id, })
+    axios({
+      method: 'GET',
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-lax/reviews?${params}`,
+      headers: { 'Authorization': credentials.TOKEN },
+    })
+    .then(res => {
+      console.log(res.data.results)
+    })
+  }
```

Comment:
.catch block here to be safe?

Reply:
yikes. This code is gone now that I refactored using hooks... sorry you read through all of it. Looks like I gotta learn how to use rebase